lib/apiservers/engine/backends/image.go

```diff
@@ -184,6 +155,7 @@ func (i *Image) PullImage(ctx context.Context, ref reference.Named, metaHeaders
 		return err
 	}
 
+	layerCache.Update()
```

Comment:
Why do you update the layer cache on images which is a read function?  Why wouldn't we update images on the pull only?